.github/semantic-release/release.config.js

```diff
@@ -1,21 +1,20 @@
 const config = {
     branches: ['master'],
-    Plugins: [
-        '@semantic-release/commit-analyzer',
-        '@semantic-release/release-notes-generator',
-        '@semantic-release/changelog',
-        '@semantic-release/github',
-        [
-            '@semantic-release/git',
+    tagFormat: "${version}",
+    plugins: [
+        ['@semantic-release/release-notes-generator',
             {
-                "assests": ["CHANGELOG.md"],
-                "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
+                preset: 'angular',
+                writerOpts: {
+                  headerPartial: '## What\'s changed',
+                  footerPartial: '\n**Full Changelog**:https://github.com/XeroAPI/Xero-OpenAPI/compare/{{previousTag}}...{{version}}'
+                }
             }
-        ],
-        ["@semantic-release/npm", {
-            "npmPublish": false,
-        }]
+         ],
+         '@semantic-release/commit-analyzer',
```

Comment:
do we require this other semantic-release plugin, it was found that these will be coming automatically right?

Reply:
I tested in my POC repo we need all these to work properly. earlier it was working on my local as I had node_modules folder present with all these packages. @vigneshk-tw